docs/standard/numerics.md

```diff
@@ -14,15 +14,15 @@ ms.assetid: dfebc18e-acde-4510-9fa7-9a0f4aa3bd11
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# Numerics in the .NET Framework
-The .NET Framework supports the standard numeric integral and floating-point primitives, as well as <xref:System.Numerics.BigInteger>, an integral type with no theoretical upper or lower bound, <xref:System.Numerics.Complex>, a type that represents complex numbers, and a set of SIMD-enabled vector types in the <xref:System.Numerics> namespace.  
-  
- In addition, System.Numerics.Vectors, the SIMD-enabled library of vectory types, was released as a NuGet package.  
+# Numerics in .NET
+
+.NET provides the standard numeric integral and floating-point primitives, as well as <xref:System.Numerics.BigInteger?displayProperty=nameWithType>, which is an integral type with no theoretical upper or lower bound, <xref:System.Numerics.Complex?displayProperty=nameWithType>, which represents complex numbers, and a set of SIMD-enabled vector types in the <xref:System.Numerics> namespace.
```

Comment:
Is just saying "standard" sufficient here? Many modern systems count `int128` (for example) as "standard", because it has special handling at the ABI layer.

Reply:
possibly instead of standard, we should say 8-bit, 16-bit, 32-bit, and 64-bit

Reply:
I would replace "standard" with "range of". Actual integer sizes are mentioned in the **Integral types** section.